packages/twig/package.json

```diff
@@ -41,15 +40,19 @@
     "@ilo-org/styles": "workspace:*",
     "@ilo-org/themes": "workspace:*",
     "@ilo-org/utils": "workspace:*",
+    "@popperjs/core": "^2.11.8",
     "video.js": "^7.21.2",
-    "videojs-youtube": "^3.0.1",
-    "@popperjs/core": "^2.11.8"
```

Comment:
@bashlk we should probably leave this in alphabetical order, no?

Reply:
I didn't rearrange the order of package, pnpm did 😄

Reply:
That's really weird. I'm not going crazy, `@` packages should come first right?